src/NUnitFramework/framework/Assert.Equality.cs

```diff
@@ -98,6 +98,44 @@ public static void AreEqual(double expected, double? actual, double delta)
 
         #endregion
 
+        #region Ints
+
+        /// <summary>
+        /// Verifies that two ints are equal. If they are not, then an
+        /// <see cref="AssertionException"/> is thrown.
+        /// </summary>
+        /// <param name="expected">The expected value</param>
+        /// <param name="actual">The actual value</param>
+        /// <param name="message">The message to display in case of failure</param>
+        /// <param name="args">Array of objects to be used in formatting the message</param>
+        public static void AreEqual(int expected, int actual, string message, params object[] args)
+        {
+            Assert.That(actual, Is.EqualTo(expected), message, args);
+        }
+        /// <summary>
+        /// Verifies that two ints are equal. If they are not, then an
+        /// <see cref="AssertionException"/> is thrown.
+        /// </summary>
+        /// <param name="expected">The expected value</param>
+        /// <param name="actual">The actual value</param>
+        /// <param name="message">The message to display in case of failure</param>
+        public static void AreEqual(int expected, int actual, string message)
```

Comment:
See @oznetmaster 's comment on the commit itself (second tab). We decided that the three argument overloads were unneeded, since the last argument is a params array.

@oznetmaster : when you comment on the commit it doesn't show up on the Files changed tab, which is where most of us do reviews. seems like an odd quirk of github, but there you are.